README.md

```diff
@@ -48,19 +48,23 @@ Now to the more technical aspects. These are not steps so much as things to keep
 
 ## Asking technical questions:
 
-Take your time, and describe as much of the context as you can manage. You certainly don’t have to assume zero knowledge, but it is really important to think about how you can make it as clear as possible to someone who might not have all the context but may still be able to help. Taking some time to think about how best to ask the question will help you to get the best sort of responses. It’s much more key than you might think, and small efforts to be thoughtful here have big flow-on effects.
+Take your time and describe as much of the context as you can manage. You certainly don’t have to assume zero knowledge, but it is really important to think about how you can make it as clear as possible to someone who might not have all the context but may still be able to help. Taking some time to think about how best to ask the question will help you to get the best sort of responses. It’s much more important than you might think, and small efforts to be thoughtful here have big flow-on effects.
 
 
 ## Code style:
 
 Keep it simple. I sound really boring saying this, but we just want nice simple code that works and that people can understand. Obviously this is within reason, and is also somewhat subjective. But the first thing to consider when writing the code is how easily it will be understood, and how will it make things easier to be build upon. Remember that this code will need to be read by other developers of different skill levels, so just writing code that works is really the baseline.  
```

Comment:
Hi @batteries76 ,
I think we it comes to explaining code style, it's best accompanied by concrete examples.

Even better - 'codify' the expectations in the form of a (set of?) linter configs, and there will be no ambiguities regarding what 'our code style' is!

Reply:
Yeah, sorry, can probably break that into two categories. 

One is the linting side of things, which we can chat about and implement. And I can add that to the document.

The other is more amorphous, and harder to codify. 
What I'm trying to avoid is complicated or 'clever' code that is hard for other people to understand or read. The sort of thing where we would like to spend three lines instead of one so that we can see what's going on. Not sure how to codify it - although I can probably find some examples:
```
processedData = await res.json().split(" ").filter(/s/).join("/").pop().etc()
```
That isn't real code, just a codish blurb to demonstrate. 
Should be at least a few lines that can help people see the transition to the final data with sensible variable names. I can add some of this into the document.

Reply:
Hi @batteries76 ,

thx for looking into the examples :bow: 
I agree setting good linting configurations/standards doesn't get us all the way there in terms of ensuring good 'code style'.

But I think the linting warnings it will give ppl quicker feedback about their code quality, which is why I think it's worth mentioning. And it's somewhat unbiased too ; either the lint checks fail, or it passes. For example, setting a maximum line length in the linter config might have caught the `processedData` example you mentioned.

Also, we want to be mindful not to make this readme lengthier than what it needs to be too! There's plenty that can be said about 'good code style', but this readme is not a blog about it! I suggest that it should be more than sufficient to offer a handful of useful links for the reader to browse through about this vast topic.